### PR TITLE
Add diagnostics with eventhub and elk template.

### PR DESCRIPTION
**Pull request by Skantha**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@Skantha__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (skkandia). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: diagnostics-eventhub-elk/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,373 @@
...
+            "tags": {
+                "displayName": "StorageAccount"
+            },
+            "properties": {
+                "accountType": "[variables('storageAccountType')]"
+            }
+        },
+
+        {
+            "apiVersion": "2015-05-01-preview",
```
Comment (singhkay) [inline diagnostics-eventhub-elk/azuredeploy.json]: Should be 2015-06-15

File: diagnostics-eventhub-elk/azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,373 @@
...
+                        }
+                    ]
+                }
+            }
+        },
+   
+        {
+            "type": "Microsoft.Compute/virtualMachines/extensions",
+            "name": "[concat(variables('vmName'),'/', variables('extensionName'))]",
+            "apiVersion": "2015-05-01-preview",
```
Comment (singhkay) [inline diagnostics-eventhub-elk/azuredeploy.json]: 2015-06-15

**Comment by singhkay** [comment]
lgtm, merged

**Pull request merged**